doc/source/cice_index.rst

```diff
@@ -110,6 +112,8 @@ section :ref:`tabnamelist`.
    "cm_to_m", "cm to meters conversion", "0.01"
    "coldice", "value for constant albedo parameterization", "0.70"
    "coldsnow", "value for constant albedo parameterization", "0.81"
+   "cona", "history units converion variable, multiplication term", "1.0"
+   "conb", "history units converion variable, additive term", "0.0"
```

Comment:
correct spelling of conversion, both lines

Reply:
good catch, I corrected it.